File: Php/00-primero/03-script.php
<?php

function repetirTexto($texto, $veces) {
    // Validaciones
    if (!is_string($texto) || !is_int($veces) || $veces <= 0) {
        return "Debes ingresar un texto válido y una cantidad de veces mayor a 0.";
    }

    // Repetir el texto
    $resultado = str_repeat($texto . ' ', $veces);
    
    // Eliminar el último espacio extra al final de la cadena
    return trim($resultado);
}

// Ejemplo de uso
echo repetirTexto("Hola mundo", 3);

?>

File: Python/00-primero/03-script.py
#!/usr/bin/env python3

def repetir_texto(texto: str, veces: int) -> str:
    """Repite un texto una cantidad de veces.

    Args:
        texto (str): La cadena de texto a repetir.
        veces (int): El número de veces que se repetirá el texto.

    Returns:
        str: El texto repetido varias veces o un mensaje de error.
    """
    
    # Validaciones
    if not isinstance(texto, str) or not isinstance(veces, int):
        raise ValueError("Debes ingresar un texto (str) y una cantidad de veces (int) válidas.")
    
    if veces <= 0:
        raise ValueError("Debes ingresar una cantidad de veces válida (mayor que 0).")
    
    # Repetir el texto usando 'join'
    return " ".join([texto] * veces)

# Ejemplo de uso
try:
    texto_final = repetir_texto("Hola mundo", 3)
    print(texto_final)
except ValueError as e:
    print(e)